### ISSUE TITLE
bug(core): ldml-const-builder fails with error "exports is not defined in ES module scope" 🙀

### ISSUE THREAD

**Issue by srl295** [keymanapp/keyman#7566]
```shell
$ core/tools/ldml-const-builder/build.sh clean build run
## [core/tools/ldml-const-builder] clean starting...
## [core/tools/ldml-const-builder] clean completed successfully
## [core/tools/ldml-const-builder] build starting...
## [core/tools/ldml-const-builder] build completed successfully
## [core/tools/ldml-const-builder] run starting...
file:///Users/srl295/src/keymanapp/keyman/core/include/ldml/ldml-const-builder/ldml-const-builder.js:7
exports.__esModule = true;
^

ReferenceError: exports is not defined in ES module scope
This file is being treated as an ES module because it has a '.js' file extension and '/Users/srl295/src/keymanapp/keyman/core/include/ldml/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
    at file:///Users/srl295/src/keymanapp/keyman/core/include/ldml/ldml-const-builder/ldml-const-builder.js:7:1
    at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:530:24)
    at async loadESM (node:internal/process/esm_loader:91:5)
    at async handleMainPromise (node:internal/modules/run_main:65:12)

Node.js v18.12.0
## [core/tools/ldml-const-builder] run failed
```

**Comment by srl295** [comment]
broke in #7328

**Comment by srl295** [comment]
looks like the const builder never was updated to es2020

**Comment by jahorton** [comment]
Pretty sure I ran into something similar with my ongoing ES Module work for Web (the 🧩line).  I think I usually resolved this via adding  `"type": "module"` to the NPM `package.json` and such.  Either way, even if it's not quite the same thing, keep me in the loop on this one.

**Comment by srl295** [comment]
@jahorton sorry for the late reply, looks like this was fixed by 2525d79715d5ba1b9c768b86665d0b78a2c300f9 in #7567

(which makes some sense given the title)
